CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: WebSocket connection will now enable compression.
 - Fixed: Calling `HAConnection.connect()` and `HAConnection.disconnect()` off the main thread no longer occasionally crashes.
 
+## [0.4] - 2024-02-01
```

Comment:
add to the unreleased above, probably

Reply:
Can I add everything in the unreleased section to 0.4 and release it afterward?

Reply:
Yeah